js/basic.js

```diff
@@ -0,0 +1,121 @@
+//Level 1
+
+/* Comments.
+    Let's start with comments - notes that people can read and computers will ignore.
+    They will help us up to guide you through the javaScript introduction journey.
+    There is a multi-line comments, like this one and also you can leave a single line comment
+    right in your code, example below. If you need to write some notes use comments starting with '//'
+*/
+
+//I'm your one-line comment
+
+
+/* Let's start with getting your code on the screen.
+    There is few ways you can do it and we will look into few of them:
+    - console.log('Hello World!'); - this line of code will print 'Hello World!' to the browser's console.
+    P.S: To see browser's console you can do right click in the window of you browser(Chrome, Firefox etc)
+    and select 'Inspect' or 'Inspect element' after that a console will appear on the bottom of the page.
+    - alert('Hello girls!'); - this line of code will pop-up a small window in your browser with text 'Hello girls!' in it,
+    but you need to refresh opened page first.
+ */
+
+
+//TODO: Now try to create an alert with any phrase you like.
+
+
+alert('hello');
+
+
+//TODO: After alert works for you, comment it out and refresh the page - it should not pop-up anymore.
+
+//TODO: What you say about trying console.log your message to browser? Send any message you like.
+
+
+console.log("I'm programming!");
```

Comment:
code itself will be deleted, as it is only to verify that everything will work in the end